src/lib/rpc/client.ts

```diff
@@ -81,6 +82,11 @@ export const getUserAccess = async (token: string) => {
   return await userServiceClient.getUserInfo({});
 };
 
+export function createProjectListClient(token: string) {
```

Comment:
Project Listing is a control / admin API. It requires tenantId and token. It should also use `cloudApiBaseUrl`